src/database/mongoClient.js

```diff
@@ -0,0 +1,55 @@
+import { MongoClient } from 'mongodb';
+
+const MONGODB_URI = process.env.MONGODB_URI;
+
+// Database Name
+const dbName = 'cofacts';
+
+export default class CofactsMongoClient {
+  /**
+   * @type {?Promise<CofactsMongoClient>}
+   */
+  static _instance = null;
+
+  static async getInstance() {
+    if (this._instance === null) {
+      this._instance = new Promise(async (resolve, reject) => {
```

Comment:
Seems that this is equivalent to:

```js
// It returns promise anyway, no need for async in this function
static getInstance() {
  // combines set-by-default and default setting
  return this._instance = this._instance || (async () => {
    const client = new CofactsMongoClient(MONGODB_URI);
    await client.mongoClient.connect();
    return client;
  })();
}
```

Moreover, naming the async arrow function properly can greatly improve its readability:

```js
// Somewhere outside the class:
//
async function instantiateClient() {
    const client = new CofactsMongoClient(MONGODB_URI);
    await client.mongoClient.connect();
    return client;
}

// Then, back to inside the class, the code is super easy to understand now:
//
static getInstance() {
  return this._instance = this._instance || instantiateClient();
}
```